### PR TITLE
Importing and serializing company data

### PR DESCRIPTION
**Pull request by Ilozuluchris**
Quick check,please am I on the right path #211

### CODE REVIEW

File: jarbas/core/management/commands/companies.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import csv
 import lzma
-
```
Comment (cuducos) [inline jarbas/core/management/commands/companies.py]: This line separating native modules imports and third-party module imports should be kept (as it's a PEP8 recommendation)

File: jarbas/core/management/commands/companies.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+from rows.fields import EmailField,DatetimeField,FloatField
```
Comment (cuducos) [inline jarbas/core/management/commands/companies.py]: Would you mind using a space after the commas (PEP8 recommendation)?

**Comment by cuducos** [comment]
I added minor comments on code style. You can use [`prospector`](https://prospector.landscape.io/) to assist you:

```console
$ pip install prospector
$ prospector -s veryhigh --max-line-length 120 jarbas
```

You can replace `jarbas` by the path of the files/modules you edited to see the report of your changes only ; )

**Comment by Ilozuluchris** [comment]
@cuducos Thanks I'll do that

**Pull request closed without merging**